build.gradle

```diff
@@ -1,3 +1,5 @@
+group = 'io.reactivex.rxswing2'
```

Comment:
Could you update this to 'io.reactivex.rxjava2' as it is the new "standard" parent project for 2.x - based components. The full id will then be `io.reactivex.rxjava2:rxswing:2.0.0-RC1`